### ISSUE TITLE
Undefined offset: 1 when using `Tracer::startActiveSpan`

### ISSUE THREAD

**Issue by maxcountryman** [DataDog/dd-trace-php#22]
Maybe I'm not using this as intended, but I cannot figure out how to use the scope abstraction without triggering this exception.

**Comment by jcchavezs** [comment]
Could you paste here your code?

**Comment by maxcountryman** [comment]
I can't paste the code directly but I can give you a high-level idea of what I'm trying to do:

```php
$tracer = new Tracer();
GlobalTracer::set($tracer);

// Previously I was using `GlobalTracer::get()->startSpan('app');` to no avail.
$globalScope = GlobalTracer::get()->startActiveSpan('app');

// Laravel-specific setup.
app()->terminating(function () {
    GlobalTracer::get()->flush();
});
```

Later on I have several event listeners--this seems to be where things go wrong:

```php
Event::listen(RequestHandled::class, function (RequestHandled $event) {
    $request = $event->request;
    $tags = [
        'host' => $request->getHost(),
        'method' => $request->method(),
    ];
    $scope = GlobalTracer::get()->startActiveSpan('request_handled');
    $span = $scope->getSpan();
    foreach ($tags as $tag => $value) {
        $span->setTag($tag, $value);
    }
    $span->finish();
    $scope->close();
});
```

Ultimately what I'd like to be able to do is differentiate parts of my trace by event type, for instance I care about the request lifecycle, db, cache, etc.

**Comment by chuck** [comment]
Fixed in master by #18
